docs/library/uasyncio.rst

```diff
@@ -158,8 +158,8 @@ class ThreadSafeFlag
 
 .. method:: ThreadSafeFlag.wait()
 
-    Wait for the flag to be set.  If the flag is already set then it returns
-    immediately.
+    Wait for the flag to be set.  If the event is already set then it returns
```

Comment:
I think it makes sense to keep the word `flag` here, to be consistent with the rest of this paragraph.